index.html

```diff
@@ -216,17 +250,20 @@ <h2>
         </dt>
         <dd>
           <p>
-            Requests that the pointer be locked to a DOM element
+            The <code>requestPointerLock(<var data-type="PointerLockOptions">options</var>)</code>
+            requests that the pointer be locked to a DOM element
             <dfn>target</dfn>. The <a>user agent</a> determines if pointer lock
```

Comment:
I don't think we want to redefine target... just link to it in the DOM spec:
 
```suggestion
            <a>potential event target</a>. The <a>user agent</a> determines if pointer lock
```

Reply:
Done and mark resolved.